.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk11
+  - openjdk17
```

Comment:
is it openjdk or correto ?

Reply:
don't know, I just continued what was there.